### ISSUE TITLE
searchApi request payload has incorrect types

### ISSUE THREAD

**Issue by albertyfwu** [HubSpot/hubspot-api-nodejs#350]
The `searchApi` request payload has the wrong types, called like `hubspotClient.crm.companies.searchApi.doSearch`. This is in the code:

```
export declare class PublicObjectSearchRequest {
    'filterGroups': Array<FilterGroup>;
    'sorts': Array<string>;
    'query'?: string;
    'properties': Array<string>;
    'limit': number;
    'after': number;
```

However, `sorts` should look like
```
{
  propertyName: string;
  direction: 'ASCENDING' | 'DESCENDING';
}
```
and `after` should be a `string`

**Comment by ksvirkou-hubspot** [comment]
https://github.com/HubSpot/hubspot-api-nodejs/issues/14

**Comment by mattcauley** [comment]
Trying to teach myself the basics of apis so I can one day build an app for our [trailer dealership](https://collinscustomcargo.com). What is the best way to learn? It took me 10 months to write my product json schema to advertise on the Google & Bing Shopping Ads. This script actually is very similar to json or is it json? Excuse me for being a newbie. I from South Georgia!

**Comment by ksvirkou-hubspot** [comment]
https://github.com/HubSpot/hubspot-api-nodejs/pull/442

**Comment by ksvirkou-hubspot** [comment]
Hi everyone.
I've changed type of  `after` to `string`  in the latest version([11.0.0-beta.0](https://www.npmjs.com/package/@hubspot/api-client/v/11.0.0-beta.0)).

**Comment by ksvirkou-hubspot** [comment]
I've published it in the latest stable version([11.0.0](https://www.npmjs.com/package/@hubspot/api-client/v/11.0.0)).
